drivers/SmartThings/philips-hue/src/init.lua

```diff
@@ -125,7 +125,7 @@ end
 ---@param device HueBridgeDevice
 local function migrate_bridge(driver, device)
   log.info_with({ hub_logs = true },
-    string.format("Migrate Bridge for device %s", (device.label or device.id or "unknown device")))
+    string.format("********************************* Migrate Bridge for device %s", (device.label or device.id or "unknown device")))
```

Comment:
Do we want this to remain in for the alpha testing?

Reply:
We can remove it, that was mostly something for me to look at go by in the live logs. We'll be filtering static log files for debugging.